webapp/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "../tsconfig.common.json",
   "compileOnSave": false,
   "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
```

Comment:
No, because if I want to use the `kyber`-src, I need this change in the tsconfig, else the kyber doesn't compile.

Try to checkout the branch, then
```bash
make src
make serve
```

With and without this change.